src/opnsense/mvc/app/controllers/OPNsense/Proxy/Api/SettingsController.php

```diff
@@ -240,4 +241,53 @@ public function fetchRBCronAction()
 
         return $result;
     }
+
+    /**
+     * get action
+     * @return array
+     */
+    public function getAction()
+    {
+        // define list of configurable settings
+        $result = array();
```

Comment:
better to call parent getAction and trust the original results, then translate the field content if it's there.